sspl_test/coverage/coverage_setup.py

```diff
@@ -57,31 +57,25 @@ def generate_cov_report(signal_number, frame):
         outfile='%scoverage/sspl_xml_coverage_report.xml')
     logger.info(f'XML coverage report generated with coverage of {cov_per} percentage.')
     ## Enable below code to inable HTML report generation
- 	# html_cov_per = co.html_report(
- 	#                     directory='%scoverage/sspl_html_coverage',
- 	#                     ignore_errors=True,
- 	#                 )
- 	# logger.info(f'HTML coverage report geverated with coverage of {html_cov_per} percentage.')\
-"""%(DATA_PATH, DATA_PATH)
+    # html_cov_per = co.html_report(
```

Comment:
There were few flake8 issues in this file, so I thought of correcting them.
But sure will mark it in this PR so that you won't waste time in understanding the change.
Can remove them as well if we really don't want them in this PR.

Reply:
in such cases, mention about flake8 upfront as review comment urself, so we can relate faster

Reply:
sure 👍